test/helpers/vmware/intg_test_context.go

```diff
@@ -109,16 +111,23 @@ func NewIntegrationTestContextWithClusters(goctx context.Context, integrationTes
 	})
 
 	var config *rest.Config
+
+	// get crd paths
+	_, filename, _, ok := goruntime.Caller(0)
+	Expect(ok).To(BeTrue(), "Failed to get information for current file from runtime")
+	root := path.Join(path.Dir(filename), "..", "..")
```

Comment:
```suggestion
	root := path.Join(path.Dir(filename), "..", "..","..")
```

Otherwise root is the "test" folder

Reply:
Slightly wondering if this entire test is either not run or it doesn't need any of the CRDs?

Reply:
Let's fix it up here best effort. But as far as I can tell we are not running all tests in CI. I couldn't find any occurrence of "Service Discovery controller integration tests" in the test or integration-test logs. Let's open a separate issue to investigate if we run all our tests in CI.

Reply:
Sure, I'll fix the path error first.